README.md

```diff
@@ -149,14 +149,57 @@ The main type of the module, responsible for all its functionality.
 *Optional.* Applies a pending reboot only for the specified reasons.
 This can take a single reason or an array of reasons.
 
-See the [Reboot when certain conditions are met](#reboot-when-certain-conditions-are-met) section for reasons why you might reboot. 
+See the [Reboot when certain conditions are met](#reboot-when-certain-conditions-are-met) section for reasons why you might reboot.
 
 #### `unless`
 
 *Optional.* Ignores the specified reasons when checking for a pending reboot.
 This can take a single reason or an array of reasons.
 
-See the [Reboot when certain conditions are met](#reboot-when-certain-conditions-are-met) section for reasons why you might reboot. 
+See the [Reboot when certain conditions are met](#reboot-when-certain-conditions-are-met) section for reasons why you might reboot.
+
+### Function: `reboot::wait`
+
+This function is intended to be used as part of a [plan](https://puppet.com/docs/bolt/0.x/writing_plans.html) and allows Bolt to wait for a server to reboot before continuing. This function has no use in normal Puppet code (outside of plans) and will not work.
```

Comment:
I'd point to 1.x or latest for docs.

Reply:
`https://puppet.com/docs/bolt/latest/writing_plans.html`

Reply:
Done